public/locales/bg/translation.json

```diff
@@ -43,8 +36,6 @@
     "required address": "Моля въведете адрес",
     "required email": "Моля въведете електронна поща",
     "increase": "Увеличи",
-    "decrease": "Намали",
-    "Sign In": "Вписване",
-    "Sign Up": "Регистрация"
-}
->>>>>>> fe73e3be52a48c55db9dec2d20d9e87fa884fc1e
+    "decrease": "Намали"
+    
```

Comment:
There is not needed extra space here and also please make sure some translation is not lost as seems more deleted lines than added.